gradio/components/textbox.py

```diff
@@ -88,6 +91,7 @@ def __init__(
             show_copy_button: If True, includes a copy button to copy the text in the textbox. Only applies if show_label is True.
             autoscroll: If True, will automatically scroll to the bottom of the textbox when the value changes, unless the user scrolls up. If False, will not scroll to the bottom of the textbox when the value changes.
             max_length: maximum number of characters (including newlines) allowed in the textbox. If None, there is no maximum length.
+            submit_btn: If False, will not show a submit button. If a string, will use that string as the submit button text.
```

Comment:
We should also mention that it removes the input field border:

<img width="392" alt="image" src="https://github.com/user-attachments/assets/cebfa3cb-b5e1-4c7b-8f2a-d76e42448930">

Reply:
i.e. compared to:

<img width="381" alt="image" src="https://github.com/user-attachments/assets/ca3a81ed-6c02-4430-8899-9e75feeea414">

Reply:
Done!